common/web/types/src/consts/virtual-key-constants.ts

```diff
@@ -144,72 +144,92 @@ export const USVirtualKeyCodes = {
-  [ k.K_Q, k.K_W, k.K_E, k.K_R, k.K_T, k.K_Y, k.K_U, k.K_I, k.K_O, k.K_P, k.K_LBRKT, k.K_RBRKT, k.K_BKSLASH ],
-  // [caps] A S D F G H J K L ; ' [enter]
-  [ k.K_A, k.K_S, k.K_D, k.K_F, k.K_G, k.K_H, k.K_J, k.K_K, k.K_L, k.K_COLON, k.K_QUOTE ],
-  // [shift] Z X C V B N M , . / [shift] *=oE2
-  [ k.K_Z, k.K_X, k.K_C, k.K_V, k.K_B, k.K_N, k.K_M, k.K_COMMA, k.K_PERIOD, k.K_SLASH ],
-  // space
-  [ k.K_SPACE ],
-];
+  0x1E: k.K_A,
+  0x1F: k.K_S,
+  0x20: k.K_D,
+  0x21: k.K_F,
+  0x22: k.K_G,
+  0x23: k.K_H,
+  0x24: k.K_J,
+  0x25: k.K_K,
+  0x26: k.K_L,
+  0x27: k.K_COLON,
+  0x28: k.K_QUOTE,
+  0x29: k.K_BKQUOTE,
 
-export const ISOVirtualKeyMap: KeyMap = [
-  // ` 1 2 3 4 5 6 7 8 9 0 - = [bksp]
-  [ k.K_BKQUOTE, k.K_1, k.K_2, k.K_3, k.K_4, k.K_5, k.K_6, k.K_7, k.K_8, k.K_9, k.K_0, k.K_HYPHEN, k.K_EQUAL ],
-  // [tab] Q W E R T Y U I O P [ ]
-  [ k.K_Q, k.K_W, k.K_E, k.K_R, k.K_T, k.K_Y, k.K_U, k.K_I, k.K_O, k.K_P, k.K_LBRKT, k.K_RBRKT  ],
-  // [caps] A S D F G H J K L ; ' \ [enter]
-  [ k.K_A, k.K_S, k.K_D, k.K_F, k.K_G, k.K_H, k.K_J, k.K_K, k.K_L, k.K_COLON, k.K_QUOTE, k.K_BKSLASH ],
-  // [shift] * Z X C V B N M , . / [shift] *=oE2
-  [ k.K_oE2, k.K_Z, k.K_X, k.K_C, k.K_V, k.K_B, k.K_N, k.K_M, k.K_COMMA, k.K_PERIOD, k.K_SLASH ],
-  // space
-  [ k.K_SPACE ],
-];
+  0x2B: k.K_BKSLASH,
+  0x2C: k.K_Z,
+  0x2D: k.K_X,
+  0x2E: k.K_C,
+  0x2F: k.K_V,
+  0x30: k.K_B,
+  0x31: k.K_N,
+  0x32: k.K_M,
+  0x33: k.K_COMMA,
+  0x34: k.K_PERIOD,
+  0x35: k.K_SLASH,
 
-export const JISVirtualKeyMap: KeyMap = [
-  // [Hankaku/Zenkaku] 1 2 3 4 5 6 7 8 9 0 - ^ ¥ [bksp]
-  [ k.K_1, k.K_2, k.K_3, k.K_4, k.K_5, k.K_6, k.K_7, k.K_8, k.K_9, k.K_0, k.K_HYPHEN, k.K_EQUAL, k.K_BKSLASH /* YEN */ ],
-  // [tab] Q W E R T Y U I O P @«`» [ [enter]
-  [ k.K_Q, k.K_W, k.K_E, k.K_R, k.K_T, k.K_Y, k.K_U, k.K_I, k.K_O, k.K_P, k.K_BKQUOTE, k.K_LBRKT  ],
-  // [caps] A S D F G H J K L ; : ] [enter]
-  [ k.K_A, k.K_S, k.K_D, k.K_F, k.K_G, k.K_H, k.K_J, k.K_K, k.K_L, k.K_COLON, k.K_QUOTE, k.K_RBRKT ],
-  // [shift] Z X C V B N M , . / _ [shift]
-  [ k.K_Z, k.K_X, k.K_C, k.K_V, k.K_B, k.K_N, k.K_M, k.K_COMMA, k.K_PERIOD, k.K_SLASH, k.K_oE2 /* ろ */ ],
-  // space
-  [ k.K_SPACE ],
-];
+  0x39: k.K_SPACE,
 
-export const ABNT2VirtualKeyMap: KeyMap = [
-  // ' 1 2 3 4 5 6 7 8 9 0 - = [bksp]
-  [ k.K_BKQUOTE, k.K_1, k.K_2, k.K_3, k.K_4, k.K_5, k.K_6, k.K_7, k.K_8, k.K_9, k.K_0, k.K_HYPHEN, k.K_EQUAL ],
-  // [tab] Q W E R T Y U I O P ´ [
-  [ k.K_Q, k.K_W, k.K_E, k.K_R, k.K_T, k.K_Y, k.K_U, k.K_I, k.K_O, k.K_P, k.K_LBRKT, k.K_RBRKT  ],
-  // [caps] A S D F G H J K L ç ~ ] [enter]
-  [ k.K_A, k.K_S, k.K_D, k.K_F, k.K_G, k.K_H, k.K_J, k.K_K, k.K_L, k.K_COLON, k.K_QUOTE, k.K_BKSLASH ],
-  // [shift] \ Z X C V B N M , . ; / [shift]
-  [ k.K_oE2, k.K_Z, k.K_X, k.K_C, k.K_V, k.K_B, k.K_N, k.K_M, k.K_COMMA, k.K_PERIOD, k.K_SLASH, k.k_oC1 /* ABNT2 */ ],
-  // space
-  [ k.K_SPACE ],
-];
+  0x56: k.K_oE2, // << Dup
+  0x73: k.k_oC1,
+  0x7D: k.K_oE2, // << Dup
```

Comment:
```suggestion
  0x56: k.K_oE2, // << Same as 0x7D; found on iso, abnt2
  0x73: k.k_oC1,
  0x7D: k.K_oE2, // << Same as 0x56; found on jis
```

We need to fix the casing of `k.k_oC1` to `k.K_oC1` but that can be a separate PR I guess.

Reply:
someother test failure so i'll pull that in…